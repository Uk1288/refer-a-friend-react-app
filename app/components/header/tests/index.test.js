import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import Header from '../index';

describe('<Header />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <Header />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
