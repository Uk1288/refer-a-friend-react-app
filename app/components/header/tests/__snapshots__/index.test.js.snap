// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match the snapshot 1`] = `
<h1
  class="header__StyledHeader-sc-1mu6x1k-0 griTSl"
>
  <span>
    Refer a friend
  </span>
</h1>
`;
